pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1137,9 +1133,9 @@ func (og *operationGenerator) GenerateUnmapVolumeFunc(
 		volumeToUnmount.VolumeSpec = csiSpec
 
 		blockVolumePlugin, err =
-			og.volumePluginMgr.FindMapperPluginByName(csi.CSIPluginName)
```

Comment:
ditto: this is just removing migration functionality. Maybe we need to comment this more clearly

Reply:
Should we consider adding more unit tests for migration to catch this?

Reply:
yes. accepting volunteers

Reply:
https://github.com/kubernetes/kubernetes/issues/77141

Reply:
Thanks for clarifying this David.